README.md

```diff
@@ -59,8 +59,9 @@ A collection of awesome email newsletter tools, platforms, media, and software.
 - [Substack Discover](https://substack.com/discover) - list of newsletters using Substack with search functionality
 - [Thanks for Subscribing](https://www.thanksforsubscribing.app/) - curated list of newsletters with topics and search functionality
 - [Viral We Grow Newsletter Listing](https://viralwegrow.com/newsletterlist) - Listing of amazing newsletters on the VWG Community and connecting to their owners
+- [Inbox Stash](https://inboxstash.com/) - Collections of amazing email newsletters on various topics, weekly digest of resources for newsletter creators and picks
```

Comment:
Can you list this in alphabetical order please?

Reply:
Ok